conformance/tests/httproute-invalid-reference-policy.go

```diff
@@ -90,7 +90,7 @@ var HTTPRouteInvalidReferencePolicy = suite.ConformanceTest{
 		// 	kubernetes.GatewayStatusMustHaveListeners(t, s.Client, gwNN, listeners, 60)
 		// })
 
-		gwAddr := kubernetes.GatewayAndHTTPRoutesMustBeReady(t, s.Client, s.ControllerName, gwNN, routeNN)
+		gwAddr := kubernetes.GatewayAndHTTPRoutesMustBeReady(t, s.Client, s.ControllerName, gwNN)
```

Comment:
Dropped the check for the route to be Accepted here pending the TODO on L49-51 to be resolved.

Reply:
Could you move the TODO from L49 down to here? I actually hadn't looked at the impl of `GatewayAndHTTPRoutesMustBeReady`, didn't realize it was actually checking the `Accepted` condition on routes.

Reply:
:+1: moved and reworded slightly